api/core/core.proto

```diff
@@ -342,6 +264,16 @@ message GetObjectResponse {
     Object object = 1;
 }
 
+message ListObjectsRequest {
+    string namespace   = 2;
```

Comment:
Nothing important, but what's the purpose of starting the numbering with 2 here? Are you planning to add some fields before `namespace` later?

Reply:
No purpose, just copy paste :tired_face: